site/content/features/recordpatterns.md

```diff
@@ -255,9 +246,9 @@ public sealed interface SubSequence extends CharSequence permits Initial, Final,
 
    default int length() {
       return switch (this) {
-         case Initial(var __, var end) -> end;
+         case Initial(var _, var end) -> end;
          case Final(var seq, var start) -> seq.length() - start();
-         case Middle(var __, var start, var end) -> end - start;
+         case Middle(var _, var start, var end) -> end - start;
```

Comment:
Below this snippet it still says:

> Note the double underscore for the variables that we don’t care about. (A single underscore is a Java keyword, held in reserve for future use.)

But I wonder whether unnamed variables should be used in this article because anyways:

a) It's another JEP
b) which i still in preview modus

Without the sandboxes could run without `preview="true"`. To avoid additional explanations in the context of thos article I would probably just use `seq` here.

Reply:
I changed all `_` to `__` and removed `preview="true"` in both articles.